sdk/build.gradle

```diff
@@ -63,7 +63,7 @@ subprojects {
         testOptions {
             unitTests {
                 all {
-                    jvmArgs '-noverify', '-Djdk.attach.allowAttachSelf=true'
+                    jvmArgs '-noverify', '-Djdk.attach.allowAttachSelf=true', '-Djdk.module.illegalAccess.silent=true'
```

Comment:
do we need this flag?

Reply:
Yes, it ignores warning that was added in new Java about black magic in PowerMock